Objects/dictobject.c

```diff
@@ -5146,13 +5199,12 @@ dictitems_contains(_PyDictViewObject *dv, PyObject *obj)
         return 0;
     key = PyTuple_GET_ITEM(obj, 0);
     value = PyTuple_GET_ITEM(obj, 1);
-    found = PyDict_GetItemWithError((PyObject *)dv->dv_dict, key);
+    if (PyDict_GetItemRef((PyObject *)dv->dv_dict, key, &found) < 0) {
```

Comment:
It can be further simplified:
```c
    result = PyDict_GetItemRef((PyObject *)dv->dv_dict, key, &found);
    if (result > 0) {
        result = PyObject_RichCompareBool(found, value, Py_EQ);
        Py_DECREF(found);
    }
    return result;
```

Reply:
Oh nice, my first implementation didn't return 1 when the item was found.

I took your code with a minor nit: ``if (result == 1)`` instead of ``if (result > 0)``.